.ansible-lint

```diff
@@ -5,3 +5,4 @@ skip_list:
    - '602'  # Don't compare to empty string
    - '701'  # Role info should contain platforms
    - '703'  # Should change default metadata
+   - 'fqcn'
```

Comment:
Leif recently asked whether we want strict fqcn enforced or not. I thought we'd already disabled it since my local ansible-lint doesn't complain (I think it might be too old).